bin/flutter

```diff
@@ -18,6 +19,7 @@ REVISION=`(cd "$FLUTTER_ROOT"; git rev-parse HEAD)`
 if [ ! -f "$SNAPSHOT_PATH" ] || [ ! -f "$STAMP_PATH" ] || [ `cat "$STAMP_PATH"` != "$REVISION" ] || [ "$FLUTTER_TOOLS_DIR/pubspec.yaml" -nt "$FLUTTER_TOOLS_DIR/pubspec.lock" ]; then
   echo Updating flutter tool...
   (cd "$FLUTTER_TOOLS_DIR"; pub get > /dev/null)
+  (cd "$FLUTTER_DIR"; pub get > /dev/null)
```

Comment:
LGTM.  I might add a comment explaining why we run pub get in this directory in case people in the future wonder